en/Advanced topics/Commercial license.md

```diff
@@ -1,53 +1,53 @@
 ### About Obsidian commercial license
 
-Obsidian is free for personal use. For commercial use in a company of at least 2 people, you need to purchase a commercial license in order to use it for work-related activities such as taking meeting notes or keeping track of a team project.
+Obsidian is free for personal use. For commercial (business) use in a company of two (2) or more employees, a commercial license is required. Business use includes any work-related activities such as capturing meeting notes, maintaining team project notes, configuring an organizational wiki, and similar note taking / document management / knowledge management functionality.
```

Comment:
Thanks for making it more consistent with the EULA to make things clearer!

Nit: I would prefer no space before and after the slash.

Reply:
The problem with removing the spaces is that you get weird line spacing, such as below. I recommend leaving in the spaces or changing slashes to commas. Let me try commas and see how that looks.

![image](https://user-images.githubusercontent.com/55888411/97935413-7901a800-1d46-11eb-822c-052ca7f8ad88.png)